applications/jupyter/variables.tf

```diff
@@ -92,21 +93,15 @@ variable "k8s_backend_service_port" {
   default     = 80
 }
 
-variable "brand" {
-  type        = string
-  description = "name of the brand if there isn't already on the project. If there is already a brand for your project, please leave it blank and empty"
-  default     = ""
-}
-
-variable "url_domain_addr" {
-  type        = string
-  description = "Domain provided by the user. If it's empty, we will create one for you."
-  default     = ""
+variable "create_brand" {
+  type        = bool
+  description = "Create Brand OAuth Screen"
+  default     = false
 }
 
-variable "url_domain_name" {
+variable "domain" {
   type        = string
-  description = "Name of the domain provided by the user. This var will only be used if url_domain_addr is not empty"
+  description = "Provide domain for ingress resource and ssl certificate. If it's empty, it will use nip.io wildcard dns"
```

Comment:
Can we add a hyperlink to nip.io so users know what it means?

We also discussed taking this as user input instead, so making the field non optional and forcing the user to type "nip.io". Ok to do this in a followup PR.

Reply:
will add in marketplace UI.. for workloads.tfvars it still valid.